pkg/k8s/configmaps/crud.go

```diff
@@ -83,7 +83,7 @@ func Create(ctx context.Context, cf *apiv1.ConfigMap, namespace string, c kubern
 	return nil
 }
 
-func update(ctx context.Context, cf *apiv1.ConfigMap, namespace string, c kubernetes.Interface) error {
+func Update(ctx context.Context, cf *apiv1.ConfigMap, namespace string, c kubernetes.Interface) error {
```

Comment:
We should probably have to use the deploy since it¡s calling the update inside and we want to check line 62

Reply:
done 7778daef659b3822220e44128db9a7119a214a70